config/manifests/remove_cert_manager_volume_patch.yaml

```diff
@@ -0,0 +1,8 @@
+# Remove the manager container's "cert" volumeMount, since OLM will create and mount a set of certs.
+# Update the indices in this path if adding or removing containers/volumeMounts in the manager's Deployment.
+- op: remove
+  path: /spec/template/spec/containers/1/volumeMounts/0
+# Remove the "cert" volume, since OLM will create and mount a set of certs.
+# Update the indices in this path if adding or removing volumes in the manager's Deployment.
+- op: remove
+  path: /spec/template/spec/volumes/0
```

Comment:
In general, I think the approach you've taken here to include this patch in a separate file is cleaner. However, in the interest of making it easy to follow future migration steps, could you instead add this inline to `config/manifests/kustomization.yaml` like the example? https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.6.1/#manifestsv2-add-a-kustomize-patch-to-remove-the-cert-manager-volumevolumemount-from-your-csv